youths/schema.py

```diff
@@ -0,0 +1,90 @@
+import graphene
+from graphene_django.types import DjangoObjectType
+from graphql_jwt.decorators import login_required
+
+from profiles.models import Profile
+
+from .consts import GENDERS, LANGUAGES
+from .models import YouthProfile
+
+
+class YouthProfileType(DjangoObjectType):
+    class Meta:
+        model = YouthProfile
+        exclude = ["id"]
+
+
+PreferredLanguage = graphene.Enum(
+    "PreferredLanguage", [(l[0].upper(), l[0]) for l in LANGUAGES]
+)
+Gender = graphene.Enum("Gender", [(g[0].upper(), g[0]) for g in GENDERS])
+
+
+class YouthProfileInput(graphene.InputObjectType):
+    ssn = graphene.String(required=True)
+    school_name = graphene.String(required=True)
+    school_class = graphene.String(required=True)
+    expiration = graphene.DateTime(required=True)
+    preferred_language = PreferredLanguage()
+    volunteer_info = graphene.String()
+    gender = Gender(required=True)
+    diabetes = graphene.Boolean()
+    epilepsy = graphene.Boolean()
+    heart_disease = graphene.Boolean()
+    extra_illnesses_info = graphene.String()
+    serious_allergies = graphene.Boolean()
+    allergies = graphene.String()
+    notes = graphene.String()
+    # approved_by = ?
+    approved_time = graphene.DateTime()
+    photo_usage_approved = graphene.Boolean()
+
+
+class UpdateYouthProfile(graphene.Mutation):
+    class Arguments:
+        youth_profile = YouthProfileInput(required=True)
+
+    youth_profile = graphene.Field(YouthProfileType)
+
+    @login_required
+    def mutate(self, info, **kwargs):
+        youth_profile_data = kwargs.pop("youth_profile")
+        profile = Profile.objects.get(user=info.context.user)
```

Comment:
Will only work if the youth profile is being created by the youth themselves, but if an admin is creating it, this would resolve as the `Profile` of the admin user, which most likely does not exist.

So I think we should probably accept an id of the youth's `Profile` or the `uuid` of the youth's `User` as input here...